server/controllers/conversation.js

```diff
@@ -0,0 +1,61 @@
+const Conversation = require("../models/Conversation");
+const User = require("../models/User");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /conversations
+// @desc Create new conversation
+//@access Private
+exports.createConversation = asyncHandler(async (req, res, next) => {
+  const { userId } = req.body;
```

Comment:
```suggestion
  const { receiverId } = req.body;
```
To avoid any confusion we can change the above to `receiverId`, as `userId` in some cases would refer to the logged in user 😄